### PR TITLE
chore(stories): update all stories to csf format

### PR DESCRIPTION
**Pull request by mainframev**
I decided to update all stories to CSF format, because in order to update to SC5 I have to solve flow errors, it will be easier to update flow first, to reduce the amount of error (i checked, some errors disappear after update) and we still have to update flow, so why not to update both. 

 Orbit.kiwi: https://orbit-docs-chore-update-all-stories-to-csf.surge.sh
 Storybook: https://orbit-chore-update-all-stories-to-csf.surge.sh

### CODE REVIEW

File: packages/orbit-components/src/Icon/Icon.stories.js [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
-          <Icon reverseOnRtl />
-        </RenderInRtl>
-      );
-    },
-    {
-      info:
-        "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
-    },
-  )
-  .lokiSkip("List of all icons", () => <IconList />);
```
Comment (silvenon) [inline packages/orbit-components/src/Icon/Icon.stories.js:68]: Let me know about this.

**Comment by silvenon** [comment]
Otherwise looks good ✅

File: packages/orbit-components/src/Icon/Icon.stories.js (hunk 1 above)
Comment (mainframev) [reply packages/orbit-components/src/Icon/Icon.stories.js:68]: fixed manually ;)

Comment (mainframev) [reply packages/orbit-components/src/Icon/Icon.stories.js:68]: updated manually

**Review by silvenon — DISMISSED** [review]

**Pull request merged**